reverse_proxy.go

```diff
@@ -97,6 +97,10 @@ var hopHeaders = []string{
 // NewSingleHostReverseProxy does not rewrite the Host header.
 // To rewrite Host headers, use ReverseProxy directly with a custom
 // director policy.
+//
+// Note: if no config.ClientOption is passed it will use the default global client.Client instance.
+// When passing config.ClientOption it will initialize a local client.Client instance.
+// If need to customize client.Client instance then use ReverseProxy.SetClient.
```

Comment:
Using ReverseProxy.SetClient if there is need for customized client.Client instance.

Reply:
done